applications/NurbsBrepApplication/custom_python/nurbs_brep_python_application.cpp

```diff
@@ -0,0 +1,45 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+
+
+// System includes
+
+#if defined(KRATOS_PYTHON)
+// External includes
+#include <pybind11/pybind11.h>
+
+
+// Project includes
+#include "includes/define.h"
+#include "nurbs_brep_application.h"
+#include "nurbs_brep_application_variables.h"
+#include "custom_python/add_custom_utilities_to_python.h"
+
+
+namespace Kratos {
+namespace Python {
+
+using namespace pybind11;
```

Comment:
I know this is done everywhere but it was recently discussed that this is bad practice and should be `using pybind as pb`, see the last comments in #2326 
Since you add a new app please use this, it is a minor change